### PR TITLE
bump: 0.2.4

### PR DESCRIPTION
**Pull request by 0xMRTT**

### CODE REVIEW

**Comment by 0xMRTT** [comment]
bot, build io.github.Bavarder.Bavarder

**Comment by 0xMRTT** [comment]
bot, build io.github.Bavarder.Bavarder

**Comment by hfiguiere** [comment]
not sure how this got merged with a failed build but the build is still failed.

File: pypi-dependencies.json [hunk 1]
```diff
@@ -12,53 +12,28 @@
...
-                    "url": "https://files.pythonhosted.org/packages/f5/ba/2b3ff0140b714149b4ee0c3af857f5e9b5fecd0d877370ea89351c21f433/baichat_py-0.2.1-py3-none-any.whl",
-                    "sha256": "af5e83d813b93b249009da35fbb6cfcea8c65ee36fa0102e3ed7340d90804391"
+                    "url": "https://files.pythonhosted.org/packages/2b/a8/050ab4f0c3d4c1b8aaa805f70e26e84d0e27004907c5b8ecc1d31815f92a/cffi-1.15.1.tar.gz",
+                    "sha256": "d400bfb9a37b1351253cb402671cea7e89bdecc294e8016a707f6d1d8ac934f9"
                 },
                 {
                     "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/ff/d7/8d757f8bd45be079d76309248845a04f09619a7b17d6dfc8c9ff6433cac2/charset-normalizer-3.1.0.tar.gz",
-                    "sha256": "34e0a2f9c370eb95597aae63bf85eb5e96826d81e3dcf88b8886012906f509b5"
+                    "url": "https://files.pythonhosted.org/packages/d1/6f/5d591a5628423af4598e2d6ff0861fcbc554cc259590fac9f97d9c984611/curl_cffi-0.5.6-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl",
```
Comment (hfiguiere) [inline pypi-dependencies.json:25]: you added an x86_64 only binary wheel which is why the build do fail.

Comment (0xMRTT) [reply pypi-dependencies.json:25]: Ohhhh thanks !!!

Comment (0xMRTT) [reply pypi-dependencies.json:25]: How can I change the url depending of the platform ?

- https://files.pythonhosted.org/packages/d1/6f/5d591a5628423af4598e2d6ff0861fcbc554cc259590fac9f97d9c984611/curl_cffi-0.5.6-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (`7a9df9fabff038f1ac9e7e6f32b5edb5d8df8c2eec64f53f513de1766c17ffdb`)
- https://files.pythonhosted.org/packages/9b/33/815c3a69b660c81447ba67b9cd7e54fc905caab2be34cc6b295a836f8b40/curl_cffi-0.5.6-cp37-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (`00e8c83b0dfde19c3470e5a07350bc8124db11723ef4e6d346cd634bb30ebc42`)

Comment (Aman9das) [reply pypi-dependencies.json:25]: https://github.com/flathub/org._2009scape.Launcher/commit/d819ad9489fcd8dcffdb6470be83be92bddfca4c

@0xMRTT

Comment (0xMRTT) [reply pypi-dependencies.json:25]: thanks !!

**Pull request merged**